tools/cli/src/mentat_cli/command_parser.rs

```diff
@@ -28,15 +27,19 @@ use combine::combinator::{
     try
 };
 
+use combine::primitives::Consumed;
+
 use errors as cli;
 
 use edn;
 
 pub static HELP_COMMAND: &'static str = &"help";
 pub static OPEN_COMMAND: &'static str = &"open";
 pub static CLOSE_COMMAND: &'static str = &"close";
-pub static QUERY_COMMAND: &'static str = &"q";
-pub static TRANSACT_COMMAND: &'static str = &"t";
+pub static QUERY_COMMAND: &'static str = &"query";
+pub static ALT_QUERY_COMMAND: &'static str = &"q";
```

Comment:
If you label these `SHORT_` and `LONG_`, your `try` clauses will read more clearly.